src/BEditaClient.php

```diff
@@ -469,6 +469,19 @@ public function schema(string $type) : ?array
         return $this->get(sprintf('/model/schema/%s', $type), null, $h);
     }
 
+    /**
+     *
+     * Get JSON SCHEMA of a relation
+     *
+     * @param string $type Object or resource type name
+     * @return array|null JSON SCHEMA in array format
+     */
+    public function relationSchema(string $type) : ?array
+    {
+        $h = ['Accept' => 'application/json'];
+	    return $this->get(sprintf('/model/relations/%s', $type), null, $h);
```

Comment:
There is something wrong with indentation. Assure to have one tab equals to 4 spaces please configuring your IDE/Editor to use `.editorconfig` file